### ISSUE TITLE
Minor issues but breaking

### ISSUE THREAD

**Issue by martianatwork** [joelbutcher/socialstream#3]
Hey,

I was using the PR branch for development and now switching to the package.

- Wrong class name in `CreateUserWithTeamsFromProvider` stub

- `connected-accounts-form.blade.php` should be
```php
$this->providers as $provider
```

instead of

```php
$this->providers as $provider => $enabled
```

- No handler for `Laravel\Socialite\Two\InvalidStateException`, to reproduce login to facebook with http, this error can be thrown for different reasons as well
- No error is shown when the user tries to login with an account which is connected to other account, ex. User1 is logged in with admin@admin.com fb account and user2 tries to login with same account

**Comment by joelbutcher** [comment]
Hey @martianatwork thanks for picking up on these issues.

The first point here is deliberate. There are two stubs with the same class name (just different file names to distinguish). If you've opted out of teams functionality on Jetstream, the other action stub gets published (check out the [`CreateNewUser`](https://github.com/laravel/jetstream/tree/1.x/stubs/app/Actions/Fortify) Fortify actions that ship with Jetstream - I used this as a basis for this type of publishing. 

The final point is a known issue, this is partly down to there being no easy mechanism for displaying errors in Jetstream V1. However, this will be fixed in Jetstream V2 with a banner at the top of the page. I'll take a look and see if there's something I can place in the form component for v1.


Thanks for the spots!

**Comment by joelbutcher** [comment]
@martianatwork The issues found above have now been sorted in the latest few merges.

I've created a new `HandleInvalidState` action that gets called whenever the exception is thrown by Socialite. All it does is throw the exception, but I've published the action into `App/Actions/Socialstream` so you're free to modify as you see fit.

Alternatively, if you wish to write your own implementation, make sure you extend `JoelButcher\Socialstream\Contracts\HandlesInvalidState` and register it in `App\Providers\SocialstreamServiceProvider`
